exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,18 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+
+function calculateGrade(mark) {
+  if(mark >= 80){
+    return "A";
+  }else if (mark >60 && mark< 80){
```

Comment:
You don't need the `mark< 80` here, because the code will only go there if the first if statement is false